test/bin/build_images.sh

```diff
@@ -29,7 +29,7 @@ extract_container_images() {
     local -r version=$1
     local -r repo_url=$2
     local -r outfile=$3
-    local -r repo_name="microshift-extract-images"
+    local -r repo_name="$(basename "${repo_url}")"
```

Comment:
The repo_url var may be the empty string at times. `basename` will fail on that and the script runs with `set -e`, shouldnt that trigger a failure? Maybe we could just get rid of the variable since it is only used in line 39?

Reply:
I can compute the name later where it's used when we know the URL has a value.